### PR TITLE
add scipy deps

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: py3-gast.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  - name: Python Build
+    runs: python setup.py build
+
+  - name: Python Install
+    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+# no versions returned from github graphql api
```
Comment (rawlingsj) [inline py3-gast.yaml:37]: I can see the tags matching the version [here](https://github.com/serge-sans-paille/gast/releases/tag/0.5.4) 

so we should be able to use
```
update:
  enabled: true
  github:
    identifier: serge-sans-paille/gast
    use-tag: true
```
which I validated using 

```
wolfictl check update  --override-version 0.0.1 py3-gast
```

File: py3-ply.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    runs: python setup.py build
+
+  - name: Python Install
+    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+# some sort of a deprecated project
+update:
+  enabled: false
```
Comment (rawlingsj) [inline py3-ply.yaml:39]: this and [related noticed](https://github.com/dabeaz/ply#important-notice---october-27-2022) may be a problem as we can only add maintained projects to wolfi

File: py3-pythran.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  - name: Python Build
+    runs: python setup.py build
+
+  - name: Python Install
+    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+# no versions returned from github graphql api
```
Comment (rawlingsj) [inline py3-pythran.yaml:41]: we should be able to use github 

```
update:
  ignore-regex-patterns:
    - .post1
  enabled: true
  github:
    identifier: serge-sans-paille/pythran
    use-tag: true
```
validated using `wolfictl check update  --override-version 0.0.1 py3-pythran`

**Review by rawlingsj — DISMISSED** [review]

File: py3-ply.yaml (hunk 2 above)
Comment (developer-guy) [reply py3-ply.yaml:39]: it might be yes but some packages depend on it "py3-pythran" to be precise

File: py3-gast.yaml (hunk 1 above)
Comment (developer-guy) [reply py3-gast.yaml:37]: I didn't use `use-tag`, let me try that again with this option.

Comment (developer-guy) [reply py3-gast.yaml]: thanks James!

File: py3-ply.yaml (hunk 2 above)
Comment (kaniini) [reply py3-ply.yaml:39]: We can package it, but we should indicate the date of the git snapshots we use.

Comment (developer-guy) [reply py3-ply.yaml:39]: thanks, I've updated it as `3.11_git20180215`

File: py3-ply.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  description: Python Lex & Yacc
+  copyright:
+    - license: BSD
+  dependencies:
+    runtime:
+      - python3
+
+var-transforms:
+  - from: ${{package.version}}
+    match: _git20180215
```
Comment (kaniini) [inline py3-ply.yaml:14]:
```suggestion
    match: _git(.*)
```

**Review by kaniini — APPROVED** [review]

**Review by kaniini — APPROVED** [review]

**Pull request merged**